app/models/operatingsystems/windows.rb

```diff
@@ -14,7 +14,7 @@ def pxe_prefix(arch)
   end
 
   def bootfile arch, type
-    pxe_prefix(arch) + eval("#{self.family}::PXEFILES[:#{type}]")
+    pxe_prefix(arch) + PXEFILES[type.to_sym]
```

Comment:
No, because in this case the family is always `Windows` - Operatingsystem.family just reads the type attribute.